cloud/scope/cluster.go

```diff
@@ -162,7 +170,7 @@ func (s *ClusterScope) setFailureDomains(ctx context.Context) error {
 	} else {
 		req := identity.ListFaultDomainsRequest{
 			CompartmentId:      common.String(s.GetCompartmentId()),
-			AvailabilityDomain: resp.Items[0].Name,
+			AvailabilityDomain: respAd.Items[0].Name,
 		}
 		resp, err := s.IdentityClient.ListFaultDomains(ctx, req)
```

Comment:
can we use that ad status here rather than using the list call again, not a big deal though

Reply:
will be resolved in https://github.com/oracle/cluster-api-provider-oci/issues/90